### PR TITLE
Fix/mute engage

### PR DESCRIPTION
**Pull request by tkhmy**
## Related Issue(required)
https://tier4.atlassian.net/browse/RT0-29602

## Description(required)
- Add mute condition for 5s

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: src/signage/src/signage/route_handler.py [hunk 1]
```diff
@@ -73,6 +74,8 @@ def __init__(
     def emergency_checker_callback(self):
         if self._parameter.ignore_emergency:
             in_emergency = False
+        if self._autoware.information.operation_mode == OperationModeState.STOP:
```
Comment (yabuta) [inline src/signage/src/signage/route_handler.py:77]: 発進のtimeoutは良さそうなのですが今回のPRでこちらの修正が入っているのはなぜでしょうか？
こちらで入れたと思ったのですが
https://github.com/tier4/AutowareSignage/pull/81/files

Comment (tkhmy) [reply src/signage/src/signage/route_handler.py:77]: rebase してないと思います。修正します。

Comment (tkhmy) [reply src/signage/src/signage/route_handler.py]: 修正しました。

**Review by yabuta — APPROVED** [review]
LGTM

**Pull request merged**